common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -47,3 +63,357 @@ def test_studio_help_links(self):
                 expected['sr_text'],
                 actual.find_element_by_xpath('following-sibling::span').text
             )
+
+
+class SignInAndSignUpHelpTest(WebAppTest):
```

Comment:
As per my understanding separate testcase classes for singin and signup would be much better.